workflow/util/util.go

```diff
@@ -150,17 +150,18 @@ func IsWorkflowCompleted(wf *wfv1.Workflow) bool {
 
 // SubmitOpts are workflow submission options
 type SubmitOpts struct {
-	Name           string                 // --name
-	GenerateName   string                 // --generate-name
-	InstanceID     string                 // --instanceid
-	Entrypoint     string                 // --entrypoint
-	Parameters     []string               // --parameter
-	ParameterFile  string                 // --parameter-file
-	ServiceAccount string                 // --serviceaccount
-	DryRun         bool                   // --dry-run
-	ServerDryRun   bool                   // --server-dry-run
-	Labels         string                 // --labels
-	OwnerReference *metav1.OwnerReference // useful if your custom controller creates argo workflow resources
+	Name             string                 // --name
+	GenerateName     string                 // --generate-name
+	InstanceID       string                 // --instanceid
+	Entrypoint       string                 // --entrypoint
+	Parameters       []string               // --parameter
+	ParameterFile    string                 // --parameter-file
+	ServiceAccount   string                 // --serviceaccount
+	DryRun           bool                   // --dry-run
+	ServerDryRun     bool                   // --server-dry-run
+	Labels           string                 // --labels
+	OwnerReference   *metav1.OwnerReference // useful if your custom controller creates argo workflow resources
+	SubstituteParams bool                   // --substitute-params
```

Comment:
Please add this to `cliSubmitOpts` in submit.go instead of here. These submit options should only be for options that will remain on the CRD (even though I see some that don't).